client/extensions/woocommerce/app/order/order-activity-log/day.js

```diff
@@ -9,7 +9,7 @@ import React, { Component } from 'react';
 /**
  * Internal dependencies
  */
-import FoldableCard from 'components/foldable-card';
+import FoldableCard from 'wcs-client/components/foldable-card';
```

Comment:
Where can I test this?

Reply:
This was introduced in https://github.com/Automattic/woocommerce-services/pull/2370#discussion_r591760370. If we want to test whether or not webpack found the right file (`client/components/foldable-card/index.js`), we could add a random class name in one of the divs and see if it renders.

Reply:
Thanks! Sorry my question was not clear. Where in the UI do I find the order-activity-log/day.js to test the changes?

Reply:
I believe it is the notes on the order page itself: 
![image](https://user-images.githubusercontent.com/572862/124195872-7b103800-da88-11eb-9d85-fee2095943d0.png)